### DISCUSSION TITLE
Confused about device models it generates automatically

### DISCUSSION THREAD

**Discussion by loganfernandes** [nVuln/homebridge-lg-thinq#124] [Q&A]
When I look at the data that it pulls in for my Fridge, Washer, Dryer, Dishwasher, and Oven (which isn't supported in this plugin) - it has all of their models completely wrong. Is there a reason for that? 

Anything I can do to help with testing/fixing that kind of thing? I also have a newer LG TV too that lives in the ThinQ app if you wanted to get that pulled in as well, I can help.

**Comment by nVuln** [comment]
model number responded via ThinQ api is different with real model number (I don't know why 🥲), nothing can help with this
if you want it showing correct model number in homekit, you can manually fill it to plugin config

TV not using ThinQ platform, so it's not covered by this plugin
but almost newest LG TV native supported homekit, no plugin needed
or you can try this plugin: [merdok/homebridge-webos-tv](https://github.com/merdok/homebridge-webos-tv)

**Reply by loganfernandes** [reply]
Ah, okay no worries then! I don’t need the models to show up exactly, just wasn’t sure if it was a real problem or not. 

That said, I also DO have an oven that I can help testing with if you still need. The request was pretty old and hadn’t been updated so I wasn’t sure if it was stale.
